tests/document_ops.rs

```diff
@@ -21,6 +21,17 @@ extern crate galvanic_assert;
 #[macro_use]
 extern crate serde_json;
 
+#[test]
+fn doc_roundtrip_empty_data() -> Result<(), IronOxideErr> {
+    let sdk = initialize_sdk()?;
+    let doc = [0u8; 0];
+
+    let doc_result = sdk.document_encrypt(&doc, &Default::default())?;
+    let decrypted_result = sdk.document_decrypt(doc_result.encrypted_data())?;
+
+    Ok(assert_eq!(&doc, decrypted_result.decrypted_data()))
```

Comment:
Is there a reason you put the assertion inside of the `Ok` for this test, but not for the other new test you added?
I think we normally just leave the `Ok(())` by itself at the end, though I guess this way is more concise.

Reply:
I did it as a single statement because there was only a single assertion at the end of the test. 
I tend to do it this way in simple cases for brevity, but either would be correct in my mind.